Dockerfile.heroku

```diff
@@ -0,0 +1,2 @@
+FROM heartexlabs/label-studio:latest
```

Comment:
If this is what Heroku always runs doesn't that mean that no matter what branch or fork you're trying to run it will always try to run the latest heartex master branch?

Maybe I'm misunderstanding how it works.

Reply:
yes, and that's exactly how it worked before. For now, it won't build from the ground up during Heroku deployment.

Reply:
Sorry does that mean it's not possible to run a fork or branch on Heroku at all, only the master branch of this repo? Can I ask why?

Reply:
No significant reason, if you think it should be the case, I return it back

Reply:
Well yes I do, otherwise it's impossible to deploy this project on Heroku with anything other than the main project's master branch, which seems like an unnecessarily harsh restriction 🤷‍♂️